spec/features/create_peep_spec.rb

```diff
@@ -1,19 +1,6 @@
 feature 'post peep' do
-  scenario 'can send a peep to database' do
-    Message.create("This is the first message")
-    Message.create("This is the second message")
-
-    visit('/')
-
-    expect(page).to have_content "This is the first message"
-    expect(page).to have_content "This is the second message"
-  end
-
   scenario 'can send a peep to the database via browser' do
-
-    visit('/peep')
-    fill_in "message", with: "May it be a light to you in dark places, when all other lights go out."
-    click_on('Submit')
+    enter_peep("May it be a light to you in dark places, when all other lights go out.")
```

Comment:
I didn't know we could do this. Something I will try and include in my feature test moving forward.